test/pool/cases/transaction.exs

```diff
@@ -4,7 +4,7 @@ defmodule Ecto.Pool.TransactionTest do
   alias Ecto.Pool
   alias Ecto.TestPool
 
-  @timeout :infinity
+  @timeout 1000
```

Comment:
Why is it :infinity? I'm guessing it's from before we had test timeouts? Test timeouts are worse because they don't show where we are deadlocked, I changed `@timeout` because it was impossible to debug with it.